src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/db/DeltakerRepository.kt

```diff
@@ -68,7 +68,7 @@ class DeltakerRepository {
             )
         },
         sistEndretAv = row.stringOrNull("na.navn") ?: row.string("d.sist_endret_av"),
-        sistEndretAvEnhet = row.stringOrNull("ne.navn") ?: row.stringOrNull("d.sist_endret_av_enhet"),
+        sistEndretAvEnhet = row.stringOrNull("d.sist_endret_av_enhet"),
```

Comment:
Er dette riktig?

Reply:
Det er en quickfix, jeg tenker at i fremtiden må vi endre dette til å ta med både navn og id, sikkert for nav-ansatt og enhet. Det blir fort vanskelig å ikke ha id hvis man trenger å gjøre oppslag igjen.

Reply:
Men.. dette virker jo nå..? Vi henter jo navenhet basert på enhetsnummer, og da får vi tilbake en enhet som ser sånn ut: 
```
data class NavEnhet(
    val id: UUID,
    val enhetsnummer: String,
    val navn: String,
)
```
Dere kan sikkert endre det, jeg forstår bare ikke hvorfor :)

Reply:
For meg er det forvirrende at det er usikkert om det er enhetsnummer eller enhetstnavn som ligger lagra.

Når vi kaller `hentEllerOpprettNavEnhet` forventes det et enhetsnummer, ikke et enhetsnavn slik jeg forstår det. Jeg får ikke tilbake en navenhet når input til `amtPersonServiceClient.hentNavEnhet(enhetsnummer)` er et enhets-navn.